include/boxfort.h

```diff
@@ -33,12 +33,14 @@
 #  ifdef BXF_BUILDING_LIB
 #   ifdef __GNUC__
 #    define BXF_API __attribute__((dllexport))
+#    define BUILDING_DLL
```

Comment:
(fyi I forgot to define BXF_BUILDING_LIB from the cmake configuration, this is now fixed on master)